cborattr/include/cborattr/cborattr.h

```diff
@@ -26,7 +26,12 @@
 #include <ctype.h>
 #include <stdio.h>
 #include <sys/types.h>
+
+#ifdef __ZEPHYR__
+#include <cbor.h>
+#else
 #include "tinycbor/cbor.h"
+#endif
```

Comment:
In the zephyr fork of tinycbor, they have made changes to include header files the mynewt way. Here is the commit which addresses that:
https://github.com/zephyrproject-rtos/tinycbor/commit/0fc68fceacd1efc1ce809c5880c380f3d98b7b6e
We should just keep what was there.